interfaces/opinion.interface.ts

```diff
@@ -2,6 +2,7 @@ import mongoose from "mongoose";
 
 interface Opinion extends mongoose.Document {
   date: Date;
+  name: string;

```

Comment:
In Figma there no "name" in opinion, also in the model there isn't "name". You can remove this field.